src/app/redux/reducers/questions.reducers.ts

```diff
@@ -0,0 +1,53 @@
+import { Action, createReducer, on, State } from '@ngrx/store';
+import { Level } from 'src/app/core/models/level.model';
+import { Question } from 'src/app/core/models/questions.model';
+
+import * as QuestionsActions from '../actions/questions.actions'
+
+export const questionsFeatureKey = 'questions';
+
+export interface QuestionsState {
+  questions: Question[];
+}
+
+export const initialState: QuestionsState = {
+  questions: [
```

Comment:
please move this data to the mock folder

Reply:
Ok! Thanks!